hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -559,6 +561,12 @@ int OtbnModel::set_no_sec_wipe_chk() {
   return 0;
 }
 
+
+int OtbnModel::disable_stack_check() {
+  disable_stack_check_flag = true;
```

Comment:
If applying my other suggestion, this changes to
```suggestion
  stack_check_enabled_ = false;
```